CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+The Babelfish project is open source and we're happy to receive external
+contributions. But, as in any licensed project, there is some important legal
+information that you need to be aware of.
+
+First, most of the components of the Babelfish project are licensed under the GPU
+v3 or the Apache License 2.0. So, in order to contribute, you must accept to license
+your contribution using either one of those licenses, depending on the subproject
+(every project should have a LICENSE file with the specific license used).
```

Comment:
That's stated in the line below, but yes, I could just reduce it to "Your contributions must be licensed under each project license as stated in the LICENSE file at the root of the project" or something like that.